pom.xml

```diff
@@ -157,4 +157,42 @@
       </extension>
     </extensions>
   </build>
+
+  <profiles>
+    <profile>
+      <id>open-source</id>
+      <build>
+        <plugins>
+          <!-- Publish artifacts to maven central instead of S3. -->
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.7</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>oss.sonatype.org</serverId>
+              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>false</autoReleaseAfterClose> <!-- TODO: Change to true. -->
```

Comment:
Leaving this set to false for the time being until we've fully tested this.  Don't want to inadvertently release anything that we don't want to go to maven central.